addons/web/static/src/legacy/js/public/public_widget.js

```diff
@@ -517,7 +517,7 @@ var RootWidget = PublicWidget.extend({
         var defs = [this._super.apply(this, arguments)];
 
         defs.push(this._attachComponents());
-        this._getRegistry().on("UPDATE", this, ({ operation, value }) => {
+        this._getRegistry().addEventListener("UPDATE", this, ({ operation, value }) => {
```

Comment:
this doesn't seem correct (specifying a parent, and not destructuring the values out of event.detail)